apps/sms/services/js/moz_mobile_message/moz_mobile_message_shim.js

```diff
@@ -0,0 +1,210 @@
+/* -*- Mode: js; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- /
+/* vim: set shiftwidth=2 tabstop=2 autoindent cindent expandtab: */
+
+
+/* global bridge, Utils */
+
+/* exported MozMobileMessageShim */
+
+'use strict';
+(function(exports) {
+
+/**
```

Comment:
Even that I used to 2-spaces indent I think it's better without indent at all :)